docs/Overview.md

```diff
@@ -0,0 +1,29 @@
+The OpenDEX Daemon ([`opendexd`](https://github.com/opendexnetwork/opendexd)) is "the node" and core of the OpenDEX network. The graphic below shows the different participants in OpenDEX and how they are connected.
+
+![](./images/participants.png)
+
+## How to Run a Node
+
+👉 as [**Liquidity Provider**](Liquidity%20Provider%20Guide.md), earning via automated arbitrage between external exchanges and OpenDEX
+
+👉 as [**Swap Provider**](Swap%20Provider%20Guide.md), sourcing liquidity on OpenDEX **(WIP)**
+
+👉 as [**Day Trader**](Day%20Trader%20Guide.md), trading instantaneously while preserving full control & privacy
+
+👉 as [**Developer**](Developer%20Guide.md), contributing or building on top of `opendexd`
```

Comment:
The current developer-guide.md is quite out of date. Maybe just point them to https://github.com/opendexnetwork/opendexd#installation

Reply:
What exactly is out of date? The main purpose of this guide is connecting a native opendexd to dependencies within opendex-docker.

Reply:
Actually, in hindsight. It should work, but I haven't tested it. 

My point was that maybe we can keep everything "development" docs in the opendexd repo install guide, but I now realize that it's 2 different guides:
- opendexd repo guide -> setup everything manually
- developer guide in docs -> use external opendexd with opendex-docker

Reply:
Correct